### ISSUE TITLE
error: no field or member function named 'addModule' in 'Build.Step.Compile'

### ISSUE THREAD

**Issue by jcs090218** [tr1ckydev/chameleon#1]
Just want to report an incompatible error in `0.12.0-dev`. It works in `0.11.0`.

```
error: no field or member function named 'addModule' in 'Build.Step.Compile'
    exe.addModule("chameleon", cham_mod);
```

Does anyone know how to "add module" in the latest master branch? 🤔

**Comment by smallkirby** [comment]
I believe this is solved by #3

**Comment by smallkirby** [comment]
I believe that it works now in Zig `0.12.0` (ref: #3), please check it again.
FYI, `Compile.addModule()` method has changed. New API is `exe.root_module.addImport()`.
